.github/workflows/coordinator.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.20.x
+        go-version: 1.21.x
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Lint
```

Comment:
don't change this, gin have bug with 1.21

Reply:
> don't change this, gin have bug with 1.21

I set the version that testcontainer depends on to 0.28.0, and there is no need to rely on golang version 1.21.